src/video/x11/SDL_x11window.c

```diff
@@ -1758,15 +1757,18 @@ void X11_SetWindowMouseGrab(SDL_VideoDevice *_this, SDL_Window *window, SDL_bool
         X11_Xinput2UngrabTouch(_this, window);
     }
     X11_XSync(display, False);
+
+    return !data->videodata->broken_pointer_grab ? 0 : SDL_SetError("The X server refused to let us grab the mouse. You might experience input bugs.");
+    ;
```

Comment:
extra semicolon?

Reply:
Actually, please change this to `if (...) {` style.

Reply:
Done